### DISCUSSION TITLE
求教，卡一个星期了

### DISCUSSION THREAD

**Discussion by tanyixiao** [apache/incubator-seata#4027] [Q&A] [answered]
2021-09-19 12:45:22.633 ERROR 1932 --- [eoutChecker_1_1] i.s.c.r.netty.NettyClientChannelManager  : no available service 'null' found, please make sure registry config correct

**Comment by wangliang181230** [comment, marked as answer]
配置问题，建议去看入门教程

**Reply by lvekee** [reply]
事务分组名没有配置正确
事务分组与高可用：http://seata.io/zh-cn/docs/user/txgroup/transaction-group-and-ha.html
事务分组介绍：http://seata.io/zh-cn/docs/user/txgroup/transaction-group.html

**Comment by tanyixiao** [comment]
问题已解决，多谢各位的帮忙

**Reply by tanyixiao** [reply]
还得跑去复制config.cof和.sh文件 
用git运行sh ${SEATAPATH}/script/config-center/nacos/nacos-config.sh -h localhost -p 8848 -g SEATA_GROUP -t 5a3c7d6c-f497-4d68-a71a-2e5e3340b3ca
将参数同步到nacos配置中心
![image](https://user-images.githubusercontent.com/80563585/134651853-4a77c1c9-7ca6-43aa-b51a-9cb65b42c2e7.png)
最后配置yml
alibaba:
      seata:
        tx-service-group: my_test_tx_group
